src/Document/Api.php

```diff
@@ -0,0 +1,217 @@
+<?php
+
+/*
+ * This file is part of the KongAdminApi package.
+ *
+ * (c) Unikorp <https://github.com/unikorp>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Unikorp\KongAdminApi\Document;
+
+use Unikorp\KongAdminApi\AbstractDocument;
+
+/**
+ * api
+ *
+ * @author VEBER Arnaud <https://github.com/VEBERArnaud>
+ */
+class Api extends AbstractDocument
+{
+    /**
+     * name
+     * @var string $name
+     */
+    protected $name = null;
+
+    /**
+     * request host
+     * @var string $requestHost
+     */
+    protected $requestHost = null;
+
+    /**
+     * request path
+     * @var string $requestPath
+     */
+    protected $requestPath = null;
+
+    /**
+     * strip request path
+     * @var boolean $stripRequestPath
```

Comment:
Symfony Coding Standards prefer to use short types (`bool` instead of `boolean`) http://symfony.com/doc/current/contributing/code/standards.html#naming-conventions